### PR TITLE
remove contact page, add contact info to homepage

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

File: app/views/pages/home.html.haml [hunk 1]
```diff
@@ -40,13 +40,24 @@
...
+        %img{alt: 'First Code Academy', src: '/assets/first_code_academy.png'}
+    .medium-12.large-6.columns
+      = link_to "https://www.makeschool.com" do
+        %img{alt: 'Make School', src: '/assets/make_school.svg'}
+      = link_to "https://www.nitrous.io/" do
+        %img{alt: 'nitrous.io', src: '/assets/nitrousio.svg'}
+      = link_to "http://www.rackspace.com/" do
+        %img{alt: 'Rackspace', src: '/assets/rackspace.svg'}
+      = link_to "https://taplytics.com/" do
+        %img{alt: 'Rackspace', src: '/assets/rackspace.svg'}
```
Comment (zachlatta) [inline app/views/pages/home.html.haml]: Should this PR add more sponsors to the homepage?

File: app/views/pages/home.html.haml [hunk 2]
```diff
@@ -80,10 +91,17 @@
...
   .row
-    %p
-      <span class="logo-h">h</span>ackEDU is generously hosted by
-      = link_to 'Rackspace', 'https://www.rackspace.com/'
-      %span.right
+    .small-12.large-4.columns
+      %p
+        We can be reached anytime at #{mail_to 'team@hackedu.us'} or 
+        at <a href="https://webchat.freenode.net/?channels=hackedu">#hackEDU</a>
+        on freenode
```
Comment (zachlatta) [inline app/views/pages/home.html.haml]: Should this have a period at the end of the sentence?

File: app/views/pages/home.html.haml [hunk 3]
```diff
@@ -80,10 +91,17 @@
...
+    .small-12.large-4.columns
+      %p
+        <span class="logo-h">h</span>ackEDU is generously hosted by
+        = link_to 'Rackspace', 'https://www.rackspace.com/'
```
Comment (zachlatta) [inline app/views/pages/home.html.haml]: Is this still necessary now that we have Rackspace listed as a sponsor on the homepage?

**Comment by MaxWofford** [comment]
I'll bring the `awaiting merge` tag back when it looks better.

File: app/views/pages/home.html.haml [hunk 4]
```diff
@@ -40,7 +40,7 @@
 .main-section.sponsors
   %h1 Sponsors
   .row.logos
-    %ul.small-block-grid-1.medium-block-grid-2
+    %ul.smallblockgrid1.mediumblockgrid2
```
Comment (zachlatta) [inline app/views/pages/home.html.haml]: Is this correct?

**Comment by MaxWofford** [comment]
@zachlatta

**Comment by zachlatta** [comment]
Can you squash 9837c3f and 13c93be?

**Comment by MaxWofford** [comment]
@zachlatta My git-fu has leveled up! I think nothing will explode if you merge this!

File: app/views/pages/home.html.haml [hunk 5]
```diff
@@ -67,12 +67,36 @@
...
+        %img{alt: 'Hack Reactor', src: '/assets/hack_reactor.png'}
+      = link_to "https://workflowy.com/" do
+        %img{alt: 'Workflowy', src: '/assets/workflowy.png'}
+      = link_to "http://firstcodeacademy.com/" do
+        %img{alt: 'First Code Academy', src: '/assets/first_code_academy.png'}
+    .medium-12.large-6.columns
+      = link_to "https://www.makeschool.com" do
+        %img{alt: 'Make School', src: '/assets/make_school.svg'}
+      = link_to "https://www.nitrous.io/" do
+        %img{alt: 'nitrous.io', src: '/assets/nitrousio.svg'}
```
Comment (zachlatta) [inline app/views/pages/home.html.haml]: Will this work with the Rails asset pipeline? You may need to use the `image_tag`.

**Comment by MaxWofford** [comment]
@zachlatta Oops! Sorry about that. I thought I had already submitted a PR for that. I've merged the image_tag patch with this branch to submit it all at once.

**Comment by zachlatta** [comment]
No worries! Is this ready for merge?

**Comment by MaxWofford** [comment]
@zachlatta It should be ready.

**Pull request merged**